src/Spd.Manager.Licence/BizProfileValidation.cs

```diff
@@ -39,7 +38,7 @@ public BizProfileUpdateRequestValidator()
             .NotEmpty()
             .When(r => r.BizTypeCode != BizTypeCode.NonRegisteredSoleProprietor && r.BizTypeCode != BizTypeCode.RegisteredSoleProprietor);
         RuleFor(r => r.BizManagerContactInfo)
-            .Must(r => r.GivenName.IsNullOrEmpty() != true &&
```

Comment:
does it behave different from .NotEmty() ?

Reply:
It can be null or empty